l10nru.properties

```diff
@@ -2,16 +2,24 @@ category=Localization
 description=Language pack for Russian
 homepageUrl=https://github.com/1c-syntax/sonar-l10n-ru
 archivedVersions=10.2
-publicVersions=9.9,10.0,10.1,10.2.1,10.3,10.4,10.5,10.6
+publicVersions=9.9,10.0,10.1,10.2.1,10.3,10.4,10.5,10.6,10.7
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-l10n-ru-plugin
 
+10.7.description=Support SonarQube 10.7
+10.7.date=2024-11-28
+10.7.sqs=[10.7,LATEST]
+10.7.sqcb=[24.12,LATEST]
+10.7.sqVersions=10.7
+10.7.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v10.7/sonar-l10n-ru-plugin-10.7.jar
+10.7.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v10.7
+
 10.6.description=Support SonarQube 10.6
 10.6.date=2024-07-12
-10.6.sqs=[10.8,LATEST]
-10.6.sqcb=[24.12,LATEST]
-10.6.sqVersions=[10.6,10.7]
+10.6.sqs=[10.6,10.6.*]
+10.6.sqcb=[24.12,24.12.*]
```

Comment:
Compatibility of public versions can't overlap. Since 24.12 will be served by your 10.7 version, you can omit this property as well.